client-sdk/ts-web/core/playground/consts.sh

```diff
@@ -0,0 +1,3 @@
+BUILD_NUMBER=3935
```

Comment:
Once we release 21.0 it would be better if these were fetched from a stable release. Maybe open an issue to change this.

Reply:
let's see if we can do that in #46. I'll file if we don't

Reply:
switched over in #46